.github/workflows/test.yml

```diff
@@ -0,0 +1,50 @@
+name: Test
+
+on:
+  push:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          sudo apt update
+          sudo apt install espeak git
+          python3 -m pip install --upgrade pip wheel cython
+          python3 -m pip install numpy
+      - name: Install TTS
+        run: |
+          python3 setup.py egg_info
+          python3 -m pip install -e .
+      - name: Lint check
+        if: github.event_name == 'pull_request'
```

Comment:
This is a direction translation of 

https://github.com/mozilla/TTS/blob/88bb20eb378f3a36c33e80551e0fe7f8c6c68869/.travis/script#L7

but I don't think limiting linting that way is smart.

Reply:
I would also suggest:

```suggestion
```

because otherwise people doing pull requests might also have to fix linting errors that were introduced in the branches itself.

Reply:
but if there is a linting error, it shouldn't pass the test anyway. So there should be no lining error on a branch that the PR  is going on.

Reply:
This assumes that the linter output is always the same. However that new linter versions can bring new checks and errors

Reply:
I am fine with applying these new checks and errors. Do you see any downside in that ?

Reply:
Direct pushes to `master`/`dev` that contain linting errors will put the CI in an error state, but at least they will be visible. I don't see downsides that should prevent this.